tests/components/coinmarketcap/test_sensor.py

```diff
@@ -35,7 +36,8 @@ def tearDown(self):
     def test_setup(self, mock_request):
         """Test the setup with custom settings."""
         with assert_setup_component(1, sensor.DOMAIN):
-            assert setup_component(self.hass, sensor.DOMAIN, {"sensor": VALID_CONFIG})
+            assert setup_component(self.hass, sensor.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)